src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -29,7 +29,8 @@
     <a href="/conversations">Conversations</a>
     <% if (request.getSession().getAttribute("user") != null) { %>
       <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else { %>
+      <a href="/logout.jsp">Logout</a>
+    <% } else{ %>
```

Comment:
It looks like some of the upstream style fixes didn't make it into your code, e.g. the spaces after `else` in `} else {`.